cluster-autoscaler/main.go

```diff
@@ -170,6 +170,7 @@ var (
 	maxInactivityTimeFlag            = flag.Duration("max-inactivity", 10*time.Minute, "Maximum time from last recorded autoscaler activity before automatic restart")
 	maxFailingTimeFlag               = flag.Duration("max-failing-time", 15*time.Minute, "Maximum time from last recorded successful autoscaler run before automatic restart")
 	balanceSimilarNodeGroupsFlag     = flag.Bool("balance-similar-node-groups", false, "Detect similar node groups and balance the number of nodes between them")
+	balanceFilterByPodsFlag          = flag.Bool("balance-filter-by-pods", true, "Enables filtering groups with similar machines by the pods that they need to fit")
```

Comment:
Please also update FAQ.md, preferably with more description of what this flag exactly does (hard to tell by just reading this description).

Reply:
reworked the PR to keep the error message but allow proceeding so we can still see that the issue exists (and remove the flag once it's fixed)